client/src/components/AuthenticatedApp.js

```diff
@@ -11,6 +11,7 @@ const AuthenticatedApp = () => {
         <Route exact path="/" component={PostTaskPage} />
         <Route exact path="/gettask" component={GetTaskPage} />
         <Route exact path="/addtask" component={PostTaskPage} />
+        <Route exact path="/start" component={Start} />
```

Comment:
Is this start page a new welcome page or it's the login/signup page??

If it's the login/signup page then this should be deleted since UnauthenticatedApp.js is the login/signup page. I can add the new Nav Bar you made to there